build/repo.targets

```diff
@@ -83,4 +93,12 @@
     <Move SourceFiles="%(NPMPackage.OutputTar)" DestinationFiles="%(NPMPackage.ArtifactPath)" />
     <Move SourceFiles="%(NPMPackage.PackageJson).bak" DestinationFiles="%(NPMPackage.PackageJson)" />
   </Target>
+
+<Target Name="PackJavaClient">
+  <Message Text="Packing Java client" Importance="high" />
+  <Exec Command="gradlew jar" WorkingDirectory="$(RepositoryRoot)clients/java/signalr" />
+  <Exec Command="gradlew sourceJar" WorkingDirectory="$(RepositoryRoot)clients/java/signalr" />
+  <Exec Command="gradlew javadocJar" WorkingDirectory="$(RepositoryRoot)clients/java/signalr" />
+  <Exec Command="gradlew generatePOM" WorkingDirectory="$(RepositoryRoot)clients/java/signalr" />
```

Comment:
I believe you can run multiple tasks by specifying them on a single command line `gradlew jar sourceJar javadocJar generatePOM`.